app/javascript/controllers/leaflet_viewer_controller.js

```diff
@@ -151,19 +152,20 @@ export default class LeafletViewerController extends Controller {
   // Add the bounding box to the map
   addBoundsOverlay(bounds) {
     const hasMapGeomValue = Object.keys(this.mapGeomValue).length > 0;
+    var defaultOptions = hasMapGeomValue ? {
+      color: '#3388ff',
+      dashArray: "5 5",
+      weight: 2,
+      opacity: 1,
+      fillOpacity: 0
+    } : {};
```

Comment:
could we extract this to a constant (either in this module or in `leaflet/constants.js` called like `DEFAULT_GEOM_OVERLAY_OPTIONS` or something like that?

Reply:
then the line would look like
`const defaultOptions = hasMapGeomValue ? DEFAULT_GEOM_OVERLAY_OPTIONS : {}`